packages/components/src/instructionList/index.js

```diff
@@ -0,0 +1,3 @@
+import InstructionList from './InstructionList';
```

Comment:
This is often used across the company, as it allows us to omit the intermediate variable:
```suggestion
export { default } from './InstructionList';
```